examples/hello_world.rs

```diff
@@ -0,0 +1,88 @@
+#![cfg_attr(windows, feature(abi_vectorcall))]
+use ext_php_rs::{constant::IntoConst, prelude::*, types::ZendClassObject};
+
+#[derive(Debug)]
+#[php_class]
+pub struct TestClass {
+    #[prop]
+    a: i32,
+    #[prop]
+    b: i32,
+}
+
+#[php_impl]
+impl TestClass {
+    #[rename("NEW_CONSTANT_NAME")]
```

Comment:
Same as `prop` macro, it's not really consistent and we could profit from the BC break to make this better IMO